src/afl-cc.c

```diff
@@ -310,6 +310,69 @@ static u8 *find_object(u8 *obj, u8 *argv0) {
 
 }
 
+void parse_fsanitize(char *string) {
+
```

Comment:
I'd probably double-check that `strlen(string) > strlen("-fsanitize=")`, here. Just because it's C...

Reply:
nah that is a given. at a minimum it is 1 byte longer than strlen("-fsanitize="), the branch which calls the function ensures that.

Reply:
Famous last words of a C developer 😛